pkg/kubelet/kubelet.go

```diff
@@ -350,6 +352,31 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 	if err != nil {
 		return nil, err
 	}
+	if len(thresholds) > 0 {
```

Comment:
So there are two sorts of configuration that directly relate to this: Kubelet configuration (eviction) and node configuration (swap). The container manager clearly validates the latter, how much do we want it to validate the former? I'm ok with passing eviction information in if we decide to do so, but this could be a slippery slope that leads to passing more and more config fields into the container manager (remember what NewMainKubelet's argument list grew into not so long ago). Perhaps we should separate validation of node configuration and kubelet configuration?